src/Components/test/E2ETest/Tests/HttpClientTest.cs

```diff
@@ -47,14 +54,14 @@ public async Task SanityCheck_ApiServerIsRunning()
             // Note that the HttpClient we're instantiating here is *not* the
             // one under test. This is not related to Blazor in any way.
             var httpClient = new HttpClient { BaseAddress = _apiServerFixture.RootUri };
-            var responseText = await httpClient.GetStringAsync("/api/greeting/sayhello");
+            var responseText = await httpClient.GetStringAsync("/subdir/api/greeting/sayhello");
```

Comment:
meta question - directed at @SteveSandersonMS as well as @javiercn - does it make sense that we're making base-paths a part of literally all of these tests?

I would expect us to have a few tests for base paths: something hello-worldey as well as some routing tests. It looks like it's **everywhere**

Reply:
Since nonempty base paths are the sort of subtle thing that could trip us up when implementing any feature that involves URLs, I think it's good that we involve them by default.

I'd certainly be happy to also have a special-case test showing that an empty base path works too.

Reply:
So routing and navigation tests then? I don't get why this is this is a concern that cross-cuts all tests.

Reply:
I'm fine with whatever way we choose to go here, but I'm not tackling that as part of this PR.